client/packages/common/src/utils/BarcodeScannerContext.test.ts

```diff
@@ -0,0 +1,49 @@
+import { parseBarcodeData, parseResult } from './BarcodeScannerContext';
+
+describe('barcode parsing', () => {
```

Comment:
Thanks, together with the link you've provided in previous PR, this makes full sense !